proto/process/agent.proto

```diff
@@ -570,35 +578,109 @@ message ServiceSessionAffinityConfig {
     int32 clientIPTimeoutSeconds = 1;
 }
 
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2107
+message Node {
+	Metadata metadata = 1;
+	NodeSpec spec = 2;
+	NodeStatus status = 3;
+	bytes yaml = 4;
+	repeated string tags = 5;
+	Host host = 6; // Used post-resolution
+}
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2312
+message NodeSpec {
+	string podCIDR = 1;
+	repeated string podCIDRs = 3;
+	string providerID = 2;
+	bool unschedulable = 4;
+	repeated Taint taints = 5;
+}
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2348
+// skipping NodeConfigStatus for now.
+message NodeStatus {
+	// Key/Value of a resource type and a corresponding quantity as an int64.
+	map<string, int64> capacity = 1;
+	// Key/Value of a resource type and a corresponding quantity as an int64.
+	map<string, int64> allocatable = 2;
+	// https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2127
+	map<string,string> nodeAddresses = 3;
+	repeated NodeCondition conditions = 4;
+	// https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2240
+	int32 daemonEndpoints = 5;
+	NodeSystemInfo nodeInfo = 6;
+	repeated string volumesInUse = 7;
```

Comment:
should we keep this now that we removed attached volumes?

Reply:
Yes, I agree. I think people are not searching that much for attached volumes that much.
I did think that it may be interesting to search for a specific volume around every node, but I am not sure what the win for the use is.

Removing it 👍